web/styles/rendered_markdown.css

```diff
@@ -884,7 +884,7 @@
         margin-top: 0;
     }
 
-    & > :last-child {
+    & > :last-child:not(.message_inline_image) {
```

Comment:
This works. I'm still not in love with having images and media participate in the size of message/preview areas in contrast to other rendered elements, but this at least fixes a longstanding bug that I've been too busy pondering to just forge ahead with a fix like this. So thanks, @N-Shar-ma!